<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWebView</class>
 <widget class="QMainWindow" name="MyWebView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MarkDownEditPyQt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_preview">
        <property name="text">
         <string>リアルタイムプレビュー</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="css_combobox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <widget class="QTextEdit" name="editor_window">
        <property name="font">
         <font>
          <family>FixedSys</family>
          <kerning>true</kerning>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWebView" name="my_webview">
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveas"/>
    <addaction name="separator"/>
    <addaction name="action_export_html"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="edit_menu">
    <property name="title">
     <string>編集(&amp;E)</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_cut"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_duplicate"/>
   </widget>
   <widget class="QMenu" name="ins_menu">
    <property name="title">
     <string>挿入(&amp;I)</string>
    </property>
    <widget class="QMenu" name="menu_ins_head">
     <property name="title">
      <string>見出し (&amp;H)</string>
     </property>
     <addaction name="action_ins_head1"/>
     <addaction name="action_ins_head2"/>
     <addaction name="action_ins_head3"/>
     <addaction name="action_ins_head4"/>
     <addaction name="action_ins_head5"/>
     <addaction name="action_ins_head6"/>
    </widget>
    <addaction name="menu_ins_head"/>
    <addaction name="action_ins_ul"/>
    <addaction name="action_ins_ol"/>
    <addaction name="action_ins_hr"/>
    <addaction name="action_ins_link"/>
    <addaction name="action_ins_idlink"/>
    <addaction name="action_ins_image"/>
    <addaction name="action_ins_cite"/>
    <addaction name="action_ins_code_multi"/>
    <addaction name="action_ins_code_word"/>
    <addaction name="action_ins_br"/>
    <addaction name="action_ins_strong"/>
    <addaction name="action_ins_em"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>表示(&amp;V)</string>
    </property>
    <addaction name="action_html_reload"/>
    <addaction name="action_preview_set"/>
    <addaction name="action_font"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="edit_menu"/>
   <addaction name="ins_menu"/>
   <addaction name="view_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveas"/>
   <addaction name="action_export_html"/>
   <addaction name="separator"/>
   <addaction name="action_ins_head1"/>
   <addaction name="action_ins_head2"/>
   <addaction name="action_ins_head3"/>
   <addaction name="action_ins_ul"/>
   <addaction name="action_ins_ol"/>
   <addaction name="action_ins_link"/>
   <addaction name="action_ins_image"/>
   <addaction name="action_ins_code_multi"/>
   <addaction name="action_ins_cite"/>
   <addaction name="action_ins_hr"/>
   <addaction name="action_ins_br"/>
   <addaction name="action_ins_strong"/>
   <addaction name="action_ins_em"/>
   <addaction name="separator"/>
   <addaction name="action_html_reload"/>
   <addaction name="separator"/>
   <addaction name="action_help"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="action_open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_fileopen.png</normaloff>:/res/icon_24x24_fileopen.png</iconset>
   </property>
   <property name="text">
    <string>開く(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>mdファイルを開く</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_save.png</normaloff>:/res/icon_24x24_save.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>mdファイルを保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_saveas">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_saveas.png</normaloff>:/res/icon_24x24_saveas.png</iconset>
   </property>
   <property name="text">
    <string>別名保存</string>
   </property>
   <property name="toolTip">
    <string>別名で保存</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_exit.png</normaloff>:/res/icon_24x24_exit.png</iconset>
   </property>
   <property name="text">
    <string>終了(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_help.png</normaloff>:/res/icon_24x24_help.png</iconset>
   </property>
   <property name="text">
    <string>ヘルプ</string>
   </property>
  </action>
  <action name="action_sampleload">
   <property name="text">
    <string>サンプル表示</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_copy.png</normaloff>:/res/icon_24x24_copy.png</iconset>
   </property>
   <property name="text">
    <string>コピー(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_paste.png</normaloff>:/res/icon_24x24_paste.png</iconset>
   </property>
   <property name="text">
    <string>貼り付け(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_cut.png</normaloff>:/res/icon_24x24_cut.png</iconset>
   </property>
   <property name="text">
    <string>切り取り(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_delete.png</normaloff>:/res/icon_24x24_delete.png</iconset>
   </property>
   <property name="text">
    <string>削除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_duplicate">
   <property name="text">
    <string>行を複製(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_ins_head2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h2.png</normaloff>:/res/icon_24x24_h2.png</iconset>
   </property>
   <property name="text">
    <string>見出し2 (&amp;2)</string>
   </property>
  </action>
  <action name="action_ins_head3">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h3.png</normaloff>:/res/icon_24x24_h3.png</iconset>
   </property>
   <property name="text">
    <string>見出し3 (&amp;3)</string>
   </property>
  </action>
  <action name="action_ins_head4">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h4.png</normaloff>:/res/icon_24x24_h4.png</iconset>
   </property>
   <property name="text">
    <string>見出し4 (&amp;4)</string>
   </property>
  </action>
  <action name="action_ins_ul">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_ul.png</normaloff>:/res/icon_24x24_ul.png</iconset>
   </property>
   <property name="text">
    <string>番号無しリスト (&amp;U)</string>
   </property>
  </action>
  <action name="action_ins_ol">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_ol.png</normaloff>:/res/icon_24x24_ol.png</iconset>
   </property>
   <property name="text">
    <string>番号付きリスト (&amp;O)</string>
   </property>
  </action>
  <action name="action_ins_hr">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_hr.png</normaloff>:/res/icon_24x24_hr.png</iconset>
   </property>
   <property name="text">
    <string>水平線 (&amp;R)</string>
   </property>
  </action>
  <action name="action_ins_link">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_link.png</normaloff>:/res/icon_24x24_link.png</iconset>
   </property>
   <property name="text">
    <string>リンク (&amp;L)</string>
   </property>
  </action>
  <action name="action_ins_image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_img.png</normaloff>:/res/icon_24x24_img.png</iconset>
   </property>
   <property name="text">
    <string>画像 (&amp;I)</string>
   </property>
  </action>
  <action name="action_ins_cite">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_cite.png</normaloff>:/res/icon_24x24_cite.png</iconset>
   </property>
   <property name="text">
    <string>引用</string>
   </property>
  </action>
  <action name="action_ins_code_word">
   <property name="text">
    <string>コード断片</string>
   </property>
  </action>
  <action name="action_config">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_config.png</normaloff>:/res/icon_24x24_config.png</iconset>
   </property>
   <property name="text">
    <string>設定</string>
   </property>
  </action>
  <action name="action_html_reload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_reload.png</normaloff>:/res/icon_24x24_reload.png</iconset>
   </property>
   <property name="text">
    <string>html表示を更新</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_ins_br">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_newline.png</normaloff>:/res/icon_24x24_newline.png</iconset>
   </property>
   <property name="text">
    <string>改行</string>
   </property>
  </action>
  <action name="action_ins_idlink">
   <property name="text">
    <string>参照リンク</string>
   </property>
  </action>
  <action name="action_ins_em">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_em.png</normaloff>:/res/icon_24x24_em.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="action_ins_strong">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_strong.png</normaloff>:/res/icon_24x24_strong.png</iconset>
   </property>
   <property name="text">
    <string>太字</string>
   </property>
  </action>
  <action name="action_ins_code_multi">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_pre.png</normaloff>:/res/icon_24x24_pre.png</iconset>
   </property>
   <property name="text">
    <string>コード複数行</string>
   </property>
  </action>
  <action name="action_preview_set">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_preview.png</normaloff>:/res/icon_24x24_preview.png</iconset>
   </property>
   <property name="text">
    <string>リアルタイムプレビュー切替</string>
   </property>
  </action>
  <action name="action_export_html">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_export.png</normaloff>:/res/icon_24x24_export.png</iconset>
   </property>
   <property name="text">
    <string>htmlエクスポート</string>
   </property>
  </action>
  <action name="action_ins_head6">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h6.png</normaloff>:/res/icon_24x24_h6.png</iconset>
   </property>
   <property name="text">
    <string>見出し6 (&amp;6)</string>
   </property>
  </action>
  <action name="action_ins_head1">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h1.png</normaloff>:/res/icon_24x24_h1.png</iconset>
   </property>
   <property name="text">
    <string>見出し1 (&amp;1)</string>
   </property>
  </action>
  <action name="action_ins_head5">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/res/icon_24x24_h5.png</normaloff>:/res/icon_24x24_h5.png</iconset>
   </property>
   <property name="text">
    <string>見出し5 (&amp;5)</string>
   </property>
  </action>
  <action name="action_font">
   <property name="text">
    <string>フォント変更</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
